remote_udfs/README.md

```diff
@@ -0,0 +1,6 @@
+# Remote UDFs
```

Comment:
Move the remote_udfs/ directory inside the udfs/ directory

Reply:
moved